### ISSUE TITLE
On Ubuntu 17.10, scrolling is stoped by "hover" of a tune

### ISSUE THREAD

**Issue by pieplu** [gnumdk/lollypop#1279]
### Environment
- Lollypop version: 0.9.304
- GNOME/GTK+ versions: Gnome 3.24
- GNU/Linux distribution: Ubuntu 17.10
- Laptop: Lenovo T460s

### Bug/Feature

The scoll is stoped when cusrsor hover a tune. We have to move the cursor on a "tuneless" zone to be able to scroll.

Using: 
- Track-point center bouton
- Trackpad

They are no bug when using a scroll roll on a mousse. I trying to change config on my trackpad, but it don't change anything.

See gif (I can send you more capture if needed):
![](https://framapic.org/QwbmndIoTbe4/KvWBgfOrYJ0L)

**Comment by gnumdk** [comment]
#1220

**Comment by gnumdk** [comment]
Welcome to Ubuntu, they ship a patch 074_eventbox_scroll_mask.patch forcing scolling on eventbox...

Report to launchpad...

**Comment by gnumdk** [comment]
Just uploaded a gtk package to my ppa fixing the issue, an apt-get upgrade should do the job ;)

**Comment by gnumdk** [comment]
https://bugs.launchpad.net/ubuntu/+source/gtk+3.0/+bug/1728617

**Comment by pieplu** [comment]
Thanks a lot! It fix the problem!

But it will be useful for other ubuntu user.... It was the straw that broke the camel, I think I will switch to Arch... ^^

**Comment by gnumdk** [comment]
@pieplu Canonical is going to remove the patch ;)

**Comment by jbicha** [comment]
The Ubuntu patch was [dropped](https://launchpad.net/ubuntu/+source/gtk+3.0/3.22.26-2ubuntu1) in Ubuntu 18.04 Alpha "Bionic". We're hesitant to drop the patch in stable releases in case some other app was dependent on the patch's behavior.

**Comment by gnumdk** [comment]
@jbicha Can't see how app can be dependent on this patch, it's not present in all other Linux distribution.

**Comment by jbicha** [comment]
@gnumdk Let me rephrase then: the Ubuntu Desktop Team doesn't currently think this issue meets our [Stable Release Update](https://wiki.ubuntu.com/StableReleaseUpdates) criteria.

**Comment by d9h20f** [comment]
"In the following cases a stable release update is also applicable as they have a low potential for regressing existing installations but a high potential for improving the user experience"

Wouldn't the removal of the reported Ubuntu patch fall under exactly those conditions? User experience is remarkably improved and no existing installations will experience any regressions.

**Comment by gnumdk** [comment]
>but a high potential for improving the user experience

Not sure a bug in Lollypop is improving the user experience as Lollypop is not packaged by Debian/Ubuntu ;)

**Comment by jbicha** [comment]
@d9h20f Thank you for your feedback. The problem is that gtk3 is a low-level library used by a *lot* of stuff. I am not familiar enough with the patch or the gtk3 codebase to know if the patch removal could cause a regression somewhere. It wasn't a problem changing this behavior early in the current development release cycle, but we are a lot more risk-averse for stable Ubuntu releases.
